### DISCUSSION TITLE
How to set the background of a sselect listview to be opaque

### DISCUSSION THREAD

**Discussion by iincity** [syf20020816/SurrealismUI#40] [Q&A]
![image](https://github.com/Surrealism-All/SurrealismUI/assets/5435797/bcafd266-8376-46be-8e6c-8624f2c84633)

![image](https://github.com/Surrealism-All/SurrealismUI/assets/5435797/009ede2e-011b-4d68-a36a-18bec1212ba1)

**Comment by syf20020816** [comment]
That is not opaque, the reason is : **the level of SSelect is lower than other components**
you need to add `z` property to control the level
## Example
```slint
SSelect {
// z is larger than SButton's z
    z: 100;
    y: 100px;
    font-weight: 700;
    is-show: true;
    theme: Error;
    card-width: 300px;
    options: [
      {label:"Shangai",value:"s01"},
      {label:"Los Angeles",value:"l02"},
      {label:"New York",value:"n03"},
      {label:"Hong Kong",value:"h04"},
    ];
    changed(index,item)=>{
      t.index = index;
      t.label = item.label;
      t.value = item.value;
      self.close();
    }
  }
  SButton {
    z: 10;
    text: "up";
  }
```

**Comment by syf20020816** [comment]
If z is larger, the level of the component will be higher
